restx-samplest/src/main/resources/specs/mongo/objectsWithIdAnnotation_persistence.spec.yaml

```diff
@@ -8,4 +8,4 @@ wts:
        POST mongo/objectsWithIdAnnotation
        {"label": "FOO"}
     then: |
-       {"_id":"5167cec5856107c479739654","label": "FOO"}
+       {"_id":null,"label": "FOO"}
```

Comment:
I'm surprised we have to change this test: isn't it spotting a problem in the upgrade?

Reply:
Yep, that's the purpose of https://github.com/bguerout/jongo/issues/325 introduced during `jongo@1.1` => `jongo@1.2` upgrade